loop.js

```diff
@@ -0,0 +1,22 @@
+// loop => for, while, do-while
+function forLoopTest() {
+  const testData = [1, "Hello", 77, 90, "Ostad", "Test loop", 55, 0, 100];
+  for (let i = 0; i <= testData.length - 1; i++) {
+    let data = testData[i];
+    console.log(data);
+  }
+}
+
+// forLoopTest();
```

Comment:
remove commented code

Reply:
We need the code for running the for loop test.